modules/osm/tags.js

```diff
@@ -152,6 +150,10 @@ export var osmOneWayTags = {
         'goods_conveyor': true,
         'piste:halfpipe': true
     },
+    'oneway': {
+        'yes': true,
+        '1': true
```

Comment:
`oneway=1` seems to be practically [extinct](https://taghistory.raifer.tech/?#***/oneway/1). Shall we just drop it?